assets/css/sensei-course-theme/content.scss

```diff
@@ -0,0 +1,23 @@
+.sensei-course-theme {
+	&__main-content {
```

Comment:
I think this probably would make more sense to be part of the theme styles (in the theme folder). But we can also tweak it later when working on the theme.

Reply:
I'm thinking we should move everything there, except for styles that a block would need on its own.

Reply:
This sounds like out of scope of this PR. But let me know if you want me to include it here and move the styles into the theme folder. 👍

Reply:
Yeah! I think it could be done in other PR when we tweak the theme styles. ;)